nf_core/lint.py

```diff
@@ -137,9 +176,9 @@ def check_pipeline (self):
             with open(os.devnull, 'w') as devnull:
                 nfconfig_raw = subprocess.check_output(['nextflow', 'config', self.path], stderr=devnull)
         except subprocess.CalledProcessError as e:
-            logging.error("Nextflow config returned non-zero error code: {},\n   {}".format(e.returncode, e.output))
+            logging.error("Nextflow config returned non-zero error code: %s,\n   %s", e.returncode, e.output)
         else:
-            logging.debug("{} lines of pipeline config found!".format(len(nfconfig_raw.splitlines())))
+            logging.debug("%s lines of pipeline config found!", len(nfconfig_raw.splitlines()))
```

Comment:
This is 'C' coding style,  unfortunately still in the Python language :D. I would prefer the `.format()` style

Reply:
Yeah, looks like C, but is pylint recommended actually

Reply:
uff, ok was not aware of that. Than I retract this comment.

Reply:
Personally I use `.format()` always for everything. It’s recommended by the “Phil’s code style guide” :laugh:

Reply:
(sorry @andreas-wilm - that sounded more snarky than intended on re-reading, that wasn't the intention! Also pre-coffee here...)

Reply:
No worries. Didn't read this as snarky comment. I also use format() for everything, but eventually gave in for logger because of pylint warnings. In that case it's actually the correct thing to do because https://stackoverflow.com/questions/29147442/how-to-fix-pylint-logging-not-lazy but we can stick to a uniform style as well